thavalon-webapp/src/components/gameComponents/root.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from "react";
+import { GameSocket, ConnectionError, OutboundMessageType, InboundMessage } from "../../utils/GameSocket";
+import { RoleInformation } from "./roleInformation";
+import { MissionResults } from "./missionResults";
+import { PlayerBoard } from "./playerBoard";
+
+import "../../styles/gameStyles/gameGlobals.scss";
+
+export function GameRoot(): JSX.Element {
+    // ComponentDidMount runs after all children render, so we can send the 
+    // request for all game snapshots here.
+    useEffect(() => {
+        // This is a hack. We should support a light and dark mode theme for this
```

Comment:
not pre-alpha needed?

Reply:
Yeah it's not. Just something we might want to fix in the future.